From 28ecd1f24f82a90852b65f57c1caa98838d25c97 Mon Sep 17 00:00:00 2001
From: Antonio Alvarez Feijoo <antonio.feijoo@suse.com>
Date: Tue, 26 Sep 2023 09:43:37 +0200
Subject: [PATCH 03/10] fix(resume): add new systemd-hibernate-resume.service

Since https://github.com/systemd/systemd/commit/a628d933, the generator only
does the initial validation of the system info and then enables the new
`systemd-hibernate-resume.service`.

Fixes #2513

(Cherry-picked a2fe89116db4b286fbf515f26bd1773b5e6ee8ad from PR#2557)
---
 modules.d/95resume/module-setup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/95resume/module-setup.sh b/modules.d/95resume/module-setup.sh
index d2551033..9204f9cb 100755
--- a/modules.d/95resume/module-setup.sh
+++ b/modules.d/95resume/module-setup.sh
@@ -44,6 +44,7 @@ install() {
     if dracut_module_included "systemd" && [[ -x $dracutsysrootdir$systemdutildir/systemd-hibernate-resume ]]; then
         inst_multiple -o \
             "$systemdutildir"/system-generators/systemd-hibernate-resume-generator \
+            "$systemdsystemunitdir"/systemd-hibernate-resume.service \
             "$systemdsystemunitdir"/systemd-hibernate-resume@.service \
             "$systemdutildir"/systemd-hibernate-resume
         return 0
-- 
2.42.0

